tests/PHPUnit/System/expected/test_reportLimiting__UserCountry.getRegion_day.xml

```diff
@@ -36,13 +36,13 @@
 	</row>
 	<row>
 		<label>Others</label>
-		<nb_uniq_visitors>12</nb_uniq_visitors>
-		<nb_visits>60</nb_visits>
-		<nb_actions>80</nb_actions>
-		<nb_users>12</nb_users>
-		<max_actions>2</max_actions>
-		<sum_visit_length>35</sum_visit_length>
-		<bounce_count>30</bounce_count>
+		<nb_uniq_visitors>30</nb_uniq_visitors>
```

Comment:
Same as above about the limit change, all these numbers are larger now so just thinking what impact could this have on user reports.